tests/unit/aiplatform/test_featurestores.py

```diff
@@ -1023,7 +1023,25 @@ def test_list_entity_types(self, list_entity_types_mock):
         aiplatform.init(project=_TEST_PROJECT)
```

Comment:
To recreate the failure here: https://github.com/googleapis/python-aiplatform/issues/1684

You need an additional test that doesn't use `init` and passes in featurestore_id, project, and location.

Reply:
Good catch Sasha, added a test for EntityType `list_features` as well that doesn't use `init()`